Aptoide/aptoide-client-v8
# [ASV-1167] remove adult content setting

**What does this PR do?**

 Removes the adult content from the cobrand (both in settings and trough the app

**Database changed?**

No

**Where should the reviewer start?**

-SettingsFragment.java

**How should this be manually tested?**

  -Navigate to Settings (cobrand) and see if there’s nothing related to settings
  -Navigate to Settings (vanilla) and see adult content it’s still there
  -Search for an adult content app (cobrand) (xxx) and check if nothing appears
  -Search for an adult content app (vanilla) (xxx) with adult content on and check if anything appears
  -Check for possible use cases that may break the app

**What are the relevant tickets?**

  https://aptoide.atlassian.net/browse/ASV-1167

**Code Review Checklist**
